NeuroID/Utilities/NIDConfigService.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  NIDConfigService.swift
+//  NeuroID
+
+import Foundation
+import Alamofire
+
+internal class NIDConfigService {
+    
+    struct ResponseData: Decodable {
+        var callInProgress: Bool = true
+        var geoLocation: Bool = true
+        var eventQueueFlushInterval: Int = 5
+        var eventQueueFlushSize: Int = 2000
+        var requestTimeout: Int = 10
+        var gyroAccelCadence: Bool = true
+        var gyroAccelCadenceTime: Int = 200
+
+        enum CodingKeys: String, CodingKey {
+            case callInProgress = "call_in_progress"
+            case eventQueueFlushInterval = "event_queue_flush_interval"
+            case eventQueueFlushSize = "event_queue_flush_size"
+            case geoLocation = "geo_location"
+            case gyroAccelCadence = "gyro_accel_cadence"
+            case gyroAccelCadenceTime = "gyro_accel_cadence_time"
+            case requestTimeout = "request_timeout"
+        }
```

Comment:
Can you confirm if there is a test case for these mappings?  I don't see a local JSON, are the requests to neuro-dev exercising this?

Reply:
Test in place that encodes with all these mappings. Would fail if mappings did not match